core/perf_test/PerfTest_ViewResize.hpp

```diff
@@ -222,6 +296,16 @@ void run_resizeview_tests7(int N, int R) {
     }
     time7 = timer.seconds() / R;
   }
+  {
+    Kokkos::View<double*******, Layout> a("A7", N2, N1, N1, N1, N1, N1, N1);
+    Kokkos::Timer timer;
+    for (int r = 0; r < R; r++) {
+      Kokkos::View<double*******, Layout> a_(a);
+      Kokkos::resize(Kokkos::view_resize(Kokkos::WithoutInitializing), a_,
+                     int(N2 * 1.1), N1, N1, N1, N1, N1, N1);
```

Comment:
Is there a reason why you always only resize the first dimension? Also did you make sure that `N1`, `N2`, etc. are large enough that `int(N2 * 1.1)` is greater than `N2`?

Reply:
@Rombur Thanks, Bruno.
I just inherited the perf test for the default resize (i.e. with initialization ) here. But I think with only the first dimension we can test both LayoutLeft (stride of 1) and LayoutRight (stride > 1) for multiple dimensions. @crtrott can you correct me if I am wrong?

I already checked int(N1*1.1) is greater than N1, int(N2*1.1) is greater than N2, etc.

Reply:
@ctrott please confirm and merge if you are happy with the PR.